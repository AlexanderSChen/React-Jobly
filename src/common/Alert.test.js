import React from "react";
import { render } from "@testing-library/react";
import Alert from "./Alert";

it("renders without crashing", function() {
    render(<Alert />);
});

it("Matches snapshot for danger", function() {
    let messages = ["Everything is broken", "Run for your life!!!"];
    const { asFragment } = render(<Alert type="danger" messages={messages} />);
    expect(asFragment()).toMatchSnapshot();
});

it("Matches snapshot for success", function() {
    let messages = ["Everything is awesome!"];
    const { asFragment } = render(<Alert type="success" messages={messages} />);
    expect(asFragment()).toMatchSnapshot();
});