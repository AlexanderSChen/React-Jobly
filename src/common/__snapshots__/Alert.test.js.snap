// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot for danger 1`] = `
<DocumentFragment>
  <div
    class="alert alert-danger"
    role="alert"
  >
    <p
      class="mb-0 small"
    >
      Everything is broken
    </p>
    <p
      class="mb-0 small"
    >
      Run for your life!!!
    </p>
  </div>
</DocumentFragment>
`;

exports[`Matches snapshot for success 1`] = `
<DocumentFragment>
  <div
    class="alert alert-success"
    role="alert"
  >
    <p
      class="mb-0 small"
    >
      Everything is awesome!
    </p>
  </div>
</DocumentFragment>
`;
